### ISSUE TITLE
Version 1.1.0

### ISSUE THREAD

**Issue by tylermilner** [BottleRocketStudios/iOS-Hyperspace#8]
Looks like we've got a good batch of changes merged into 'master'. Let's plan to cut a 1.1.0 release by end of week.

Here's what I've gathered for a changelog so far. Let me know if there's anything I'm missing!

**1.1.0 Changelog**
* Adjusted project structure to better support Travis-CI. CI is fully up-and-running on all 3 currently supported platforms. **Carthage** is now required to work on the library and run the example projects (**Cocoapods** is no longer used). Clone the repo, run `carthage update`, and then open `Hyperspace.xcworkspace` to get started. Fixes #3.
* Can now `cancelAllTasks()` on `BackendServiceProtocol`.
* `NetworkService` logic extracted into `NetworkServiceHelper` to better support future framework compatibility with `URLSessionDelegate` implementations.
* `NetworkService` can now represent loading via the global network activity indicator shown in the device's status bar. Just initialize your `NetworkService` with a `NetworkActivityIndicatable` (ex: `NetworkService(networkActivityIndicatable: UIApplication.shared)`).
* Improved code coverage > 90%. Fixes #1.
* Re-enabled SwiftLint. Fixes #9.

**PENDING**
* MacOS Support. - #6

Release checklist:
- [ ] Update Podspec (podspec version and git tag)
- [ ] Validate Readme
- [ ] Reverse commit f7281549a4c5ad2fa16f15a62cfd344c3bdaa24d (to re-enable `pod spec lint` command on Travis-CI)

**Comment by wmcginty** [comment]
@tylermilner Do we want to wait on macOS support for 1.1 or move forward with 1.1 and put that in a tentative 1.2 release?

**Comment by tylermilner** [comment]
@wmcginty I'm cool with releasing 1.1 without macOS support. We can always do a quick 1.2 fast-follow with macOS support.

**Comment by wmcginty** [comment]
I believe as soon as #16 is merged, we're good to tag and push to trunk.

**Comment by wmcginty** [comment]
[1.1.0](https://github.com/BottleRocketStudios/iOS-Hyperspace/releases/tag/1.1.0) has been pushed to trunk.
